shell/aliases/rails.sh

```diff
@@ -122,6 +122,9 @@ function psql
   if [[ "$(__database_yml adapter)" == 'postgresql' ]]; then
     PGDATABASE="$(__database_yml database)" command psql "$@"
     return $?
+  elif [[ "$(__database_yml url)" =~ '^postgres' ]]; then
+    command psql "$(__database_yml url)" "$@"
+    return $?
   fi
   command psql "$@"
 }
```

Comment:
With both `url` and `adapter`+`database` defined in `database.yml`, Rails seems to prefer `url`. I feel like we should do the same.

Interestingly, it seems `url` in `database.yml` is preferred over the `DATABASE_URL` environment variable. I expected `DATABASE_URL` to always take preference.

What about something like this?

```sh
function psql
{
   if [[ "$(__database_yml url)" =~ ^postgres ]]; then
     command psql "$(__database_yml url)" "$@"
     return $?
   elif [[ "$DATABASE_URL" =~ ^postgres ]]; then
     command psql "$DATABASE_URL" "$@"
     return $?
   elif [[ "$(__database_yml adapter)" == 'postgresql' ]]; then
     PGDATABASE="$(__database_yml database)" command psql "$@"
     return $?
   fi
   command psql "$@"
 }
```

Reply:
Love it!